package/gluon-config-mode-mesh-vpn/luasrc/lib/gluon/config-mode/wizard/0300-mesh-vpn.lua

```diff
@@ -46,31 +46,31 @@ return function(form, uci)
 		end
 	end
 
-	o = s:option(Value, "limit_ingress", pkg_i18n.translate("Downstream (kbit/s)"))
+	o = s:option(Value, "limit_ingress", pkg_i18n.translate("Downstream (Mbit/s)"))
 	o:depends(limit, true)
-	o.default = uci:get("simple-tc", "mesh_vpn", "limit_ingress")
+	o.default = uci:get("simple-tc", "mesh_vpn", "limit_ingress") / 1024
```

Comment:
The site.conf parameters for TC are optional. The need_number-function will return nil and the parameter will thus not be added to the uci configuration in /lib/gluon/upgrade/500-mesh-vpn in that case which will result in uci:get returning nil. A division of a nil value will result in an error.

You need to check the return value first.
`o.default = uci:get("simple-tc", "mesh_vpn", "limit_ingress") or 0
o.default = o.default / 1024`

Another option is to make a default value for TC mandatory in site.conf which makes sense as we need to implicitly set a default value here.